app/views/wizard/steps/import_dates/show.html.erb

```diff
@@ -3,7 +3,7 @@
 <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters">
-      <%= form_for @import_date, builder: GOVUKDesignSystemFormBuilder::FormBuilder, url: import_date_path do |f| %>
+      <%= form_for @import_date, builder: GOVUKDesignSystemFormBuilder::FormBuilder, url: import_date_path, html: { novalidate: true } do |f| %>
```

Comment:
I'm assuming we're skipping validations that are done automatically here? Not sure I understand this change

Reply:
It's something that GDS wants.